### ISSUE TITLE
Segfault when saving study after completion

### ISSUE THREAD

**Issue by bmenn** [kubeflow/katib#97]
I have not looked into why this might be occurring, but I came across this segfault error trying to run `examples/random-cpu.yml` and `examples/mnist-cpu.yml`.

katib checkout at: ecb27de9e7a0d2ffbf8715635c6a7d4517f1602e
katib-cli: katib-cli-darwin-amd64 v0.1.1-alpha

Logs:
```
2018/05/25 18:40:03 Worker: dlk
2018/05/25 18:41:33 Study mnist is already exist (Project ID 1)
2018/05/25 18:41:33 Study w06b53328cefce8c start.
2018/05/25 18:41:33 Study conf name:"mnist" owner:"root" optimization_type:MAXIMIZE parameter_configs:<configs:<name:"--lr" parameter_type:DOUBLE feasible:<max:"0.07" min:"0.03" > > configs:<name:"--num-epochs" parameter_type:INT feasible:<max:"20" min:"10" > > > suggest_algorithm:"random" suggestion_parameters:<name:"SuggestionNum" value:"2" > suggestion_parameters:<name:"MaxParallel" value:"2" > objective_value_name:"Validation-accuracy" metrics:"accuracy" metrics:"Validation-accuracy" image:"mxnet/python" command:"python" command:"/mxnet/example/image-classification/train_mnist.py" scheduler:"default-scheduler"
2018/05/25 18:41:34 Created Lt s7a3421ec7f226d8.
2018/05/25 18:41:34 Created Lt r2ab0e99a5c1588c.
2018/05/25 18:42:10 Trial r2ab0e99a5c1588c is completed.
2018/05/25 18:42:10 Objective Value: 0.970442
panic: runtime error: invalid memory address or nil pointer dereference [signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x10621a9]

goroutine 70 [running]:
main.(*server).saveCompletedModels(0xc4201b28a0, 0xc420346870, 0x10, 0xc4202e06e0, 0xc42018c9e0, 0x0)
        /go/src/github.com/kubeflow/katib/cmd/manager/main.go:91 +0x699
main.(*server).trialIteration(0xc4201b28a0, 0xc4202e06e0, 0xc420346870, 0x10, 0xc4203b6000, 0xc4203b6060, 0x0, 0x0)
        /go/src/github.com/kubeflow/katib/cmd/manager/main.go:158 +0xf0a
created by main.(*server).CreateStudy
        /go/src/github.com/kubeflow/katib/cmd/manager/main.go:236 +0x5b9
```

**Comment by YujiOshima** [comment]
@bmenn Thank you for your report.
There are drastic changes in the manager from v0.1.1-alpha.
We need to release a new version.
I  think this bug is fixed in new version.

**Comment by bmenn** [comment]
Thanks for the update on the new changes. I will try to rebuild and see if the error has disappeared.

**Comment by gaocegege** [comment]
We released v0.1.2-alpha, please have a try :smile:

**Comment by jlewi** [comment]
Closing this bug since we have a much newer version so this bug is presumably no longer relevant.
